devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,23 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu: MediaTek MT7628AN
+cpu_cores: 1
+cpu_mhz: 580
+memory_mb: 64
+storage:
+  - type: unknown
+    size_mb: 8
+device_type: WiFi Router
+ethernet_100m_ports: 1
+jtag: false
+serial: 57600 / 8N1
+outdoor: false
+supported_current_rel: none
+supported_since_rel: snapshot
```

Comment:
These two would require regular updates; in this case I have a strong preference for dropping them. My idea of these files would be to not require any updates. Released devices can be tracked in other ways.

Reply:
I already deleted them from the validation as I had the same thought. Just forgot this file.

Reply:
Removed.